cosmic/src/evolv2.f

```diff
@@ -2642,20 +2643,65 @@ SUBROUTINE evolv2(kstar,mass,tb,ecc,z,tphysf,
 * Decide between accreted mass by secondary and/or system mass loss.
 *
          taum = mass(j2)/dm1*tb
-         if(kstar(j2).le.2.or.kstar(j2).eq.4)then
+         
+
+*
+* KB 4/Jan/21: adding in acc_lim flags
+* acc_lim = 0: standard BSE w/ MS/HG/CHeB assumed to have thermal
+*              limit of 10*taum while giants are unlimited
+* acc_lim = -1: MS/HG/CHeB assumed to have thermal
+*               limit of taum while giants are unlimited
+* acc_lim = -2: accretor kw=0-6 have thermal limit of 10*taum
+* acc_lim = -3: accretor kw=0-6 have thermal limit of taum
+* acc_lim > 0: fraction of donor mass loss accreted
 *
-* Limit according to the thermal timescale of the secondary.
+* Note that acc_lim > 0 means that the thermal limit is not considered
 *
+
+         if(acc_lim.eq.0)then
+            if(kstar(j2).le.2.or.kstar(j2).eq.4)then
+                dm2 = MIN(1.d0,10.d0*taum/tkh(j2))*dm1
+            elseif(kstar(j2).eq.3.or.kstar(j2).eq.5.or.
+     &             kstar(j2).eq.5)then
```

Comment:
Is this supposed to be `kstar(j2).eq.6`?

Reply:
same as below -- good catch!

Reply:
This is fixed in commit: 056e207